examples/EvaluationDataToApplicationInsights/appsettings.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ApplicationInsights": {
+    "ConnectionString": "<connection-string>"
+  },
+  "FeatureManagement": {
+    "FeatureA": {
+      "TelemetryEnabled": true,
+      "EnabledFor": [
+        {
+          "Name": "Targeting",
+          "Parameters": {
+            "Audience": {
+              "Users": [
+                "ross"
```

Comment:
Editing the username cookie in the browser. Perhaps I should just exclude individual targeting from the example.

Reply:
> Also, doesn't seem like having a hardcoded user already included may be desirable.

I wanted to see what happened for the not logged-in case. We could adjust it, but I think it's nice to show events flowing for guest users.